samples/25.message-reaction/src/main/java/com/microsoft/bot/sample/messagereaction/MessageReactionBot.java

```diff
@@ -0,0 +1,58 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.microsoft.bot.sample.messagereaction;
+
+import com.microsoft.bot.builder.ActivityHandler;
+import com.microsoft.bot.builder.MessageFactory;
+import com.microsoft.bot.builder.TurnContext;
+import com.microsoft.bot.schema.Activity;
+import com.microsoft.bot.schema.MessageReaction;
+import java.util.List;
+import java.util.concurrent.CompletableFuture;
+
+
+public class MessageReactionBot extends ActivityHandler {
+	private static final ActivityLog log;
```

Comment:
This should not be `static`, it's just `final` as [C#](https://github.com/microsoft/BotBuilder-Samples/blob/main/samples/csharp_dotnetcore/25.message-reaction/Bots/MessageReactionBot.cs#L14) is `readonly`